sched/scheduler/task_scheduler_test.go

```diff
@@ -288,7 +310,7 @@ func Test_TaskAssignments_PriorityStages(t *testing.T) {
 		nodes = append(nodes, fmt.Sprintf("node%d", i))
 	}
 	testCluster := makeTestCluster(nodes...)
-	cs := newClusterState(testCluster.nodes, testCluster.ch, nil)
+	cs := newClusterState(testCluster.nodes, testCluster.ch, nil, statsReceiver)
```

Comment:
You could have a utility function newFinagleStats() or similar, placed inline here and delete lines 275-276. This is a small amount of boilerplate that nonetheless appears a lot and introduces a lot of noise.